packages/wrapper/test/wrapper.test.ts

```diff
@@ -57,4 +57,20 @@ describe('Test MonacoEditorLanguageClientWrapper', () => {
             await wrapper.initAndStart(config, document.getElementById('monaco-editor-root'));
         }).rejects.toThrowError('init was already performed. Please call dispose first if you want to re-start.');
     });
+
+    test('Test if ', async () => {
```

Comment:
It look like a good idea to test if, indeed (:

Reply:
😆 oh man thanks, updated ⬇️